<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>825</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="icon_with_lbl_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>803</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="sidebar_lbl">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Side Bar</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="home_large_btn">
           <property name="text">
            <string>Home</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/home-outline.svg</normaloff>:/icons/resources/home-outline.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="files_large_btn">
           <property name="text">
            <string>Files</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/list.svg</normaloff>:/icons/resources/list.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="people_large_btn">
           <property name="text">
            <string>Students</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/people.svg</normaloff>:/icons/resources/people.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="records_large_btn">
           <property name="text">
            <string>Records</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/bookmark.svg</normaloff>:/icons/resources/bookmark.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>559</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_large_btn">
         <property name="text">
          <string>Exit</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/resources/close-circle.svg</normaloff>:/icons/resources/close-circle.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>845</width>
        <height>803</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="menu_btn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/resources/ellipsis-vertical.svg</normaloff>:/icons/resources/ellipsis-vertical.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="prev_btn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>28</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;&lt; Previous</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="lesson_linedit">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>28</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>Lesson 1</string>
             </property>
             <property name="cursorPosition">
              <number>8</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="next_btn">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>28</height>
              </size>
             </property>
             <property name="text">
              <string>Next &gt;&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="minimumSize">
            <size>
             <width>820</width>
             <height>500</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(230, 230, 230);</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="home_page_0">
            <property name="minimumSize">
             <size>
              <width>820</width>
              <height>500</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="home_lbl">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>25</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Home</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="files__page_1">
            <property name="minimumSize">
             <size>
              <width>820</width>
              <height>500</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="file_lbl">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>25</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Files</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="students_page_2">
            <property name="minimumSize">
             <size>
              <width>820</width>
              <height>500</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="student_lbl">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>25</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Students</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="records_page_3">
            <property name="minimumSize">
             <size>
              <width>820</width>
              <height>500</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="records_lbl">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>25</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Records</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>803</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="logo_lbl">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/logo/resources/rsz_default.jpg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="home_small_btn">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/home-outline.svg</normaloff>:/icons/resources/home-outline.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="files_small_btn">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/list.svg</normaloff>:/icons/resources/list.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="people_small_btn">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/people.svg</normaloff>:/icons/resources/people.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="records_small_btn">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/resources/bookmark.svg</normaloff>:/icons/resources/bookmark.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>559</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_small_btn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/resources/close-circle.svg</normaloff>:/icons/resources/close-circle.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>home_small_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_large_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>files_small_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>files_large_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>people_small_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>people_large_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>home_large_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_small_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>files_large_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>files_small_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>people_large_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>people_small_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_small_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>797</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>829</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_large_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>782</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>819</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>records_small_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>records_large_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>records_large_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>records_small_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_with_lbl_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
